tests/unit/DispatchServiceValidation.unitTest.ts

```diff
@@ -0,0 +1,84 @@
+import {ITarget} from "../../src/models";
+import {Configuration} from "../../src/utils/Configuration";
+import {DispatchService} from "../../src/services/DispatchService";
+import * as testResult from "../resources/demoTestResult.json";
+
+describe("isValidMessageBody", () => {
+  beforeAll(() => {
+    jest.clearAllMocks();
+    jest.resetAllMocks();
+    jest.restoreAllMocks();
+  });
+  const target: ITarget = {
+    queue: "",
+    dlQueue: "",
+    swaggerSpecFile: "API_Vehicle_Testresults_28.1.yaml",
+    schemaItem: "completeTestResults",
+    endpoints: {
+      INSERT: "",
+      MODIFY: "",
+      REMOVE: "",
+    }
+  };
+  describe("When validation = true", () => {
+    let secretMock: any;
+    afterEach(() => {
+      jest.clearAllMocks();
+    });
+    afterAll(() => {
+      secretMock.mockRestore();
+    });
+    beforeAll(() => {
+      secretMock = jest.spyOn(Configuration.prototype, "getSecretConfig").mockResolvedValue(Promise.resolve({
+        baseUrl: "",
+        apiKey: "",
+        validation: "true"
+      }));
+    });
+
+    it("returns false when evaluating a completely invalid record against a valid spec", async () => {
+      const svc = new DispatchService(new (jest.fn()), new (jest.fn()));
+      const output = await svc.isValidMessageBody({something: "invalid"}, target);
+      expect(secretMock).toHaveBeenCalled();
+      expect(output).toEqual(false);
+    });
+    // it("returns true when evaluating a 'good' record against a valid spec", async () => {
```

Comment:
I think you should enable this test

Reply:
Has now been fixed. Was temporarily broken.